vm/src/stdlib/io.rs

```diff
@@ -442,16 +442,71 @@ fn text_io_wrapper_init(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
 fn text_io_base_read(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     arg_check!(vm, args, required = [(text_io_base, None)]);
 
+    let io_module = vm.import("_io", &vm.ctx.new_tuple(vec![]), 0)?;
+    let buffered_reader_class = vm
+        .get_attribute(io_module.clone(), "BufferedReader")
+        .unwrap()
+        .downcast::<PyClass>()
+        .unwrap();
```

Comment:
All of this can just be

```rust
let buffered_reader_class = vm.try_class("_io", "BufferedReader")?;
```

Reply:
Nice, have missed it after all the refactors since March when I last looked at that :D.